src/UICircularProgressRing/UICircularProgressRing.swift

```diff
@@ -133,6 +133,9 @@ fileprivate extension CALayer {
                 print("Attempted to set a value less than minValue, value has been set to minValue.\n")
                 value = minValue
             }
+            if value > maxValue {
```

Comment:
Could you add a print like above?
Also, could you change it for this and the one above to only print in debug mode.
```swift
#if debug
// print statements
#endif
```